src/FilterList.js

```diff
@@ -23,6 +26,23 @@ export let Label = inject(_.pick('tree'))(
       {tree &&
         node && (
           <span className="filter-field-label-icon">
+            {!node.updating &&
+              !!node.hasValue &&
+              tree.disableAutoUpdate &&
+              // find if any nodes in the tree are marked for update (i.e. usually nodes are marked for update because they react to "others" reactor)
+              _.some(
```

Comment:
Did you try just checking tree.markedForUpdate? That should probably do it I think

Reply:
@daedalus28 I thought so too but tree.markedForUpdate remains false no matter how many times I update other filter nodes. Not sure if this is a contexture-client bug or something else?

Reply:
Actually, that makes sense, this part is probably fine. We should consider adding something for this in contexture-client but this is ok for now